boot/storage.txt

```diff
@@ -0,0 +1,424 @@
+    pages.
+    
+    +------------------+-------------------+-----------------------------------+
+    |    Filesystem    |   Command         |   Userspace Utilities (Repo)      |
+    +------------------+-------------------+-----------------------------------+
+    |                  |                   |                                   |
+    |    btrfs         |   mkfs.btrfs      |   btrfs-progs (Community)         |
+    |                  |                   |                                   |
+    |    ext2          |   mkfs.ext2       |   e2fsprogs (Extra)               |
+    |                  |                   |                                   |
+    |    ext3          |   mkfs.ext3       |   e2fsprogs (Extra)               |
+    |                  |                   |                                   |
+    |    ext4          |   mkfs.ext4       |   e2fsprogs (Extra)               |
+    |                  |                   |                                   |
+    |    f2fs          |   mkfs.f2fs       |   f2fs-tools                      |
+    |                  |                   |                                   |
+    |    jfs           |   mkfs.jfs        |   jfsutils                        |
+    |                  |                   |                                   |
+    |    reiserfs      |   mkfs.reiserfs   |   reiserfsprogs                   |
+    |                  |                   |                                   |
+    |    xfs           |   mkfs.xfs        |   xfsprogs                        |
+    |                  |                   |                                   |
+    |    vfat          |   mkfs.vfat       |   dosfstools                      |
+    |                  |                   |                                   |
+    |    NTFS          |   mkfs.ntfs       |   ntfs-3g (Community)             |
+    |                  |                   |                                   |
+    |    swap          |   mkswap          |                                   |
+    |                  |                   |                                   |    
+    +------------------+-------------------+-----------------------------------+
+
+
+    [3.3] Swap File
+    ____________________________________________________________________________    
+    
+    As an alternative to creating an entire partition, a swap file offers the 
+    ability to vary its size on-the-fly, and is more easily removed altogether. 
+    This may be especially desirable if disk space is at a premium.
+
+    To create a swap file, use dd commmand and specify the file size (remember 
+    to change 512, for 512MB, with the appropriate desired swap file size):
+    
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   dd if=/dev/zero of=/swapfile bs=1M count=512 status=progress           |
```

Comment:
`progress` is not a valid choice for `status` if ``dd -> busybox` - it's only valid for `coreutils dd`.